platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -209,28 +194,37 @@ - (void)startWithQueue:(dispatch_queue_t)queue completionHandler:(MGLMapSnapshot
                 dispatch_queue_t workQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
                 dispatch_async(workQueue, ^{
 #if TARGET_OS_IPHONE
-                    MGLSnapshotAttributionOptions *option = [self attributionOptionsForSize:mglImage.size attributionInfo:infos];
-                    UIImage *logoImage = option.logoImage;
+                    MGLAttributionInfoStyle attributionInfoStyle = MGLAttributionInfoStyleLong;
+                    for (NSUInteger styleValue = MGLAttributionInfoStyleLong; styleValue >= MGLAttributionInfoStyleShort; styleValue--) {
+                        attributionInfoStyle = [self styleFromValue:styleValue];
```

Comment:
`-styleFromValue:` doesn’t appear to do anything except return the passed-in value, so this line would have no effect. If the compiler is warning about passing an `NSUInteger` into -`attributionSizeWithLogoStyle:sourceAttributionStyle:`, then cast `styleValue` to an `MGLAttributionInfoStyle`, because we know it’s going to be a valid value.